app/renderer/actions/Inspector.js

```diff
@@ -71,6 +71,11 @@ const uniqueAttributes = [
  */
 function xmlToJSON (source) {
   let xmlDoc;
+  
+  // Replace strings with Unicode format &#012345 with #012345
+  // The &# unicode format breaks the parser
+  source = source.replace(/&#([0-9]*)/g, "#$1");
```

Comment:
this regexp is going to replace all occurrences of `&#`. Shall we replace `[0-9]*` with `[0-9]{4,}`?